server/src/main/java/org/apache/iotdb/db/engine/storagegroup/TsFileProcessor.java

```diff
@@ -85,7 +88,7 @@
 import java.util.concurrent.locks.ReentrantReadWriteLock;
 
 @SuppressWarnings("java:S1135") // ignore todos
-public class TsFileProcessor {
+public class TsFileProcessor implements WALSubmitter {
```

Comment:
Why TsFileProcessor implements WALSubmitter? It seems so weird..

Reply:
Already deleted.